checks/evaluation/pinned_dependencies.go

```diff
@@ -316,21 +192,6 @@ func updatePinningResults(dependencyType checker.DependencyUseType,
 	pr[dependencyType] = p
 }
 
-func generateTextUnpinned(rr *checker.Dependency) string {
-	if rr.Type == checker.DependencyUseTypeGHAction {
-		// Check if we are dealing with a GitHub action or a third-party one.
-		gitHubOwned := fileparser.IsGitHubOwnedAction(rr.Location.Snippet)
-		owner := generateOwnerToDisplay(gitHubOwned)
-		return fmt.Sprintf("%s not pinned by hash", owner)
-	}
-
-	return fmt.Sprintf("%s not pinned by hash", rr.Type)
-}
-
-func generateTextIncompleteResults(e checker.ElementError) string {
-	return fmt.Sprintf("Possibly incomplete results: %s", e.Err)
-}
-
 func generateOwnerToDisplay(gitHubOwned bool) string {
 	if gitHubOwned {
```

Comment:
so this code now exists in two places (here and the probe). Not sure this needs to be dealt with right away, but long term might want to clean this up.

Reply:
i'd rather stick it in some `internal` folder if that's the goal.

Reply:
> honestly it's small enough, I think we can deal with the temporary duplication

Alright. Closing.